### ISSUE TITLE
vehiclewheel don't work correctly

### ISSUE THREAD

**Issue by Q3dlaXpoaQ** [godotengine/godot#82517]
### Godot version

4.1.1

### System information

win10 godot4.1.1 GLES3

### Issue description

When I use godot4, I found that my vehiclewheel didn't work correctly. It sometimes flies everywhere , sometimes don't work. 
When I ues godot3, it can run correctly.

### Steps to reproduce

I use the same step in godot3 to create a 3d car, but it can run in godot3 or can't run in godot4.

### Minimal reproduction project

[1.zip](https://github.com/godotengine/godot/files/12762792/1.zip)

**Comment by Calinou** [comment]
This is probably a configuration issue. VehicleWheel works correctly in the [Truck Town demo](https://github.com/godotengine/godot-demo-projects/tree/master/3d/truck_town). I advise you check how that demo project's vehicle scenes are configured.

**Comment by mathphye** [comment]
Recently I tried VehicleNode3D and also had a bad moment. At first, it is counterintuitive and it requires to "randomly" change parameters until it kind of works... 

I have tested this file, and it looks like a bad collider was assigned. Probably came from `mesh>"create single convex collision sibling"` from a mesh that was escalated, thus it doesn't match what is visually the mesh, which gives a wrong perspective of which is the body of the vehicle.

Also, the origin of mass is too high. **I think it must be shown in the viewport with a blue dot, so it is completely clear where is located.**

Assigning a lower mass origin and changing the size instead of scale, looks like solve temporally the problem. However, the VehicleBody3D shows easy inestability.

**Comment by Zireael07** [comment]
Scaled collision shapes are known to not to work in most physics nodes.

**Comment by mathphye** [comment]
Additional observations, the center of mass is not the VehicleBody3D origin as the documentation says:

> Note: The origin point of your VehicleBody3D will determine the center of gravity of your vehicle. To make the vehicle more grounded, the origin point is usually kept low, moving the CollisionShape3D and MeshInstance3D upwards.

It is from the collider child...

Also, the collider child shape somehow affects the rotational inertia, which causes the vibration where the shape has fewer length.

**An error** for my point of view, is that the distribution of forces is not distributed, then when the center of mass is not exactly in the center, it would generate a disproportional torque to its length (which is the cause of the vehicle getting away from the viewport with a jump).

in other words: When the force is equal in all wheels but the "varignon" length is shorter on one side by non-perfect-center-of-mass, it would cause a torque that makes the vehicle "jump" which is actually a really big angle momentum.

*Some like that, can't explain better by now.

**Comment by mathphye** [comment]
For example here when there is a gap between the wheels and the collider it gets "bad behavior". But it works well when the collider shape gets close to the wheels.

![image](https://github.com/godotengine/godot/assets/146298448/48eea691-9abd-4fc3-8918-fa6944953c7b)

**Comment by Q3dlaXpoaQ** [comment]
Thanks, but after I changed the center of mass, it can worked, but when I changed something others, the vehiclewheel embedded in the ground.
![Godot_v4 1 1-stable_mono_win64_BFIWmBjDWd](https://github.com/godotengine/godot/assets/129849156/ebd2914c-7bc8-4c13-8c19-8e68c3e661a8)
![Godot_v4 1 1-stable_mono_win64_pJOxcgFiKe](https://github.com/godotengine/godot/assets/129849156/895b350d-f445-4db0-882b-3d6d6ea3df6f)

Then I tried many arguments, but it either embedded in the ground, or the mass was too front or too rear.

**Comment by mathphye** [comment]
Theoretically, it should work for any center of mass positioned "inside" the four wheels (and it doesn't). However that is not the problem here, I think.

If the center of mass is out of the rectangle (z=-5) formed by the 4 wheels it would end up side down, making some wheels not grounded.

---
Also, for high mass you should increase stiffness, otherwise, the spring won't have enough force to make the vehicle stand up from the ground.

**Comment by Q3dlaXpoaQ** [comment]
> Theoretically, it should work for any center of mass positioned "inside" the four wheels (and it doesn't). However that is not the problem here, I think.
> 
> If the center of mass is out of the rectangle (z=-5) formed by the 4 wheels it would end up side down, making some wheels not grounded.
> 
> Also, for high mass you should increase stiffness, otherwise, the spring won't have enough force to make the vehicle stand up from the ground.

Thank you very much! It was so cool that it run correctly. I wonder that if I need to simulate tire attenuation and control the friction force, which argument should I change?
